modules/admin/__init__.py

```diff
@@ -117,9 +117,9 @@ def __init__(self, sh, testparam=''):
         mysuburl = ''
         if suburl != '':
             mysuburl = '/' + suburl
-        ip = get_local_ipv4_address()
+        ip = Utils.get_local_ipv4_address()
         self._port = self.mod_http._port
-        # self.logger.warning('port = {}'.format(self._port))
+        # self.logger.warning('port = {}'.format(self.port))
```

Comment:
Die Änderung verstehe ich nicht
Abgesehen davon, dass die Zeile auskommentiert ist, müsste dort self._port stehen. Wieso hast Du das inself.port geändert?

Reply:
In meinen ursprünglichen Commits stand überall self.port; beim rebase stand _port, das habe ich dann jeweils manuell angepasst. Dieses eine habe ich wohl übersehen.